src/meshes/t8code_mesh.jl

```diff
@@ -7,8 +7,6 @@ to manage trees and mesh refinement.
 """
 mutable struct T8codeMesh{NDIMS, RealT <: Real, IsParallel, NDIMSP2, NNODES} <:
                AbstractMesh{NDIMS}
-    cmesh       :: Ptr{t8_cmesh} # cpointer to coarse mesh
-    scheme      :: Ptr{t8_eclass_scheme} # cpointer to element scheme
```

Comment:
Why are these not required anymore?

Reply:
They were never anywhere required in Trixi and are for interal use within t8code.

Reply:
IIRC, they can be obtained from the `forest` pointer

Reply:
Exactly. They can be obtained via the t8code API if needed. Also the freeing of `cmesh` and `scheme` is done automatically when calling `t8_forest_unref`. So no need to keep these two around in Trixi.